src/game/overworld-init/initGamePiece.js

```diff
@@ -1,43 +1,58 @@
+const util = require('../../helpers/utilFunctions')
+
 const initGamePiece  = ( x, y, cellSize, type ) => {
+    return new gamePiece(  x, y, cellSize )
```

Comment:
in principe is dit nu een functie die een functie aanroept (fair enuf een constructor maar toch) lijkt me enigszins redundant misschien